test/release/examples/primers/nothingVariables.chpl

```diff
@@ -1,32 +1,39 @@
-// Void variables and fields
+// The 'nothing' type
 
 /*
-   This primer demonstrates the use of the ``void`` type on variables and
-   class/record fields. A variable or field that is of type ``void`` is removed
-   by the compiler and doesn't exist during program execution.
+   This primer demonstrates the use of the ``nothing`` type on variables and
+   class/record fields. A variable or field that is of type ``nothing`` is
+   removed by the compiler and doesn't exist during program execution.
+
+   The ``nothing`` type is not to be confused with the ``void`` type, which
```

Comment:
This might be highlighted more with a `.. warning::` or `.. note::`.

Also, Do we have documentation on the `void` type which we could point to there?

Reply:
I moved it into a `.. note::` section.

There is documentation for `void` in the Types section of the spec.  Should a primer be pointing into the language spec?

Reply:
I don't think so. I'll add `void` to https://github.com/chapel-lang/chapel/issues/12615 for now. Thanks